src/common/components/cards/card-kebab-menu-button.tsx

```diff
@@ -51,6 +51,7 @@ export class CardKebabMenuButton extends React.Component<CardKebabMenuButtonProp
         return (
             <div className={kebabMenuButton}>
                 <ActionButton
+                    ariaLabel="More actions"
```

Comment:
We recently updated the label on a different menu button to include the term "menu"; maybe this should be "more actions menu" or similar instead? @ferBonnin, thoughts?

Reply:
JAWS reads it as "More actions button menu" and NVDA reads it as "More actions menu button subMenu". I think NVDA shouldn't read "subMenu" before it has been opened but I'm not sure about this.

Reply:
NVDA behaviour is odd, but if it already reads more actions button menu then its fine :)